### ISSUE TITLE
unbalanced quotes

### ISSUE THREAD

**Issue by abondis** [savoirfairelinux/mod-influxdb#27]
when shinken sends an event to influxdb I always get the error message 'unbalanced quotes'

    [1438808575] ERROR: [broker-master] [influxdb broker] 400: unable to parse 'EVENT,event_type=NOTIFICATION,host_name=host1,service_description=_self_ contact="admin",notification_method="notify-host-by email",notification_type="HOST",output="Host down': unbalanced quotes                                                                                    
    [1438808575] ERROR: [broker-master] [influxdb broker] Sending data Failed. Buffering state : 11 / 300

any idea why ? I can see output doesn't have the right quotes, but I can't find in the code what would do that. I tried something like output="{}".format(data['output']) but no luck.

**Comment by titilambert** [comment]
```
[{'fields': {'alert_type': u'SERVICE',
   'attempts': 1,
   'output': u'Traceback (most recent call last):\n',
   'state': u'WARNING',
   'state_type': u'HARD',
   'time': 1438875384},
  'measurement': 'EVENT',
  'tags': {'event_type': u'ALERT',
   'host_name': u'shinken',
   'service_description': u'service:shinken-web'},
  'time': 1438875384},
 {'fields': {'event_type': 'ALERT',
   'output': u'Traceback (most recent call last):',
   'state': 'WARNING',
   'state_type': 'HARD'},
  'measurement': 'EVENT',
  'tags': {'host_name': u'shinken',
   'service_description': u'service:shinken-web'},
  'time': 1438875382},
 {'fields': {'acknowledged': 0,
   'last_check': 1438875382,
   'last_state_change': 1438875384.280424,
   'output': u'Traceback (most recent call last):',
   'state': 1,
   'state_type': 'HARD'},
  'measurement': 'SERVICE_STATE',
  'tags': {'host_name': u'shinken',
   'service_description': u'service:shinken-web'},
  'time': 1438875382},
 {'fields': {'acknownledgement': None,
   'contact': u'admin',
   'notification_method': u'notify-service-by-email',
   'notification_type': u'SERVICE',
   'state': u'WARNING',
   'time': 1438875384},
  'measurement': 'EVENT',
  'tags': {'event_type': u'NOTIFICATION',
   'host_name': u'shinken',
   'service_description': u'service:shinken-web'},
  'time': 1438875384},
 {'fields': {'acknowledged': 0,
   'last_check': 1438875384,
   'last_state_change': 1438790459.247844,
   'output': u'Agent alive and reachable',
   'state': 0,
   'state_type': 'HARD'},
  'measurement': 'HOST_STATE',
  'tags': {'host_name': u'shinken', 'service_description': ''},
  'time': 1438875384},
 {'fields': {'acknowledged': 0,
   'last_check': 1438875384,
   'last_state_change': 1438873046.247578,
   'output': u'',
   'state': 0,
   'state_type': 'HARD'},
  'measurement': 'SERVICE_STATE',
  'tags': {'host_name': u'shinken', 'service_description': u'System'},
  'time': 1438875384}]
```

**Comment by abondis** [comment]
0.9.1 and/or python-influxdb update seems to work

**Comment by aviau** [comment]
This should be fixed with influxdb 0.9.2 and influxdb-python 2.8.0.

Thanks!
